envoy-control-core/src/main/java/pl/allegro/tech/servicemesh/envoycontrol/SimpleCache.java

```diff
@@ -371,12 +374,6 @@ protected void respondWithSpecificOrder(T group,
                     String version = snapshot.version(watch.request().getResourceType(), watch.request().getResourceNamesList());
 
                     if (!watch.request().getVersionInfo().equals(version)) {
-                        if (LOGGER.isDebugEnabled()) {
-                            LOGGER.debug("responding to open watch {}[{}] with new version {}",
-                                id,
-                                String.join(", ", watch.request().getResourceNamesList()),
-                                version);
-                        }
```

Comment:
why is it getting removed?

Reply:
it produces enormous amount of logs while not adding any useful information and it actually makes it much more difficult to debug